### PR TITLE
Make space. Fixes #564

### LINKED ISSUE #564: Remove banner on event details page
**As**
user of the portal 

**I do not want to**
see the banner for each event

**in order to**
be not disturbed by the pattern that does not match to the theme of the event.

### PR DESCRIPTION
**Pull request by tom95**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -345,8 +345,8 @@ DEPENDENCIES
   jquery-turbolinks
   jquery-ui-rails
   less-rails
-  owlcarousel-rails!
   mini_magick
+  owlcarousel-rails!
```
Comment (tom95) [inline Gemfile.lock]: since this is supposed to be alphabetical some prior pr must have been wrong on this

**Review by bjrne — APPROVED** [review]
Code approved 🍂

File: app/assets/stylesheets/events.css [hunk 2]
```diff
@@ -3,6 +3,10 @@
   They will automatically be included in application.css.
 */
 
+h1.event-title {
```
Comment (radscheit) [inline app/assets/stylesheets/events.css:6]: We worked on the same, see my current PR :)

**Comment by tom95** [comment]
Is superseded by @radscheit 's PR

**Pull request closed without merging**